src/bundle/Resources/public/js/scripts/core/collapse.js

```diff
@@ -18,4 +130,29 @@
             collapseNode.dataset.collapsed = false;
         });
     });
-})(window, window.document);
+
+    if (toggleAllBtns) {
+        toggleAllBtns.forEach((btn) => {
+            btn.addEventListener('click', (event) => {
+                event.stopPropagation();
+
+                const collapseAll = btn?.querySelector('.ibexa-multi-collapse__toggler-collapse');
```

Comment:
In what case will this button be null/undefined? Do we need optional chaining here?
```suggestion
                const collapseAll = btn.querySelector('.ibexa-multi-collapse__toggler-collapse');
```